GDPR/GDPRManager.swift

```diff
@@ -67,50 +62,69 @@ public class GDPRManager {
                                                       showTermsSwitch: true)
         return ConfirmationView(viewModel: confirmationViewModel!, onConfirm: onConfirm)
     }
-    
+
     // MARK: - Helper Functions
-    
+
     public func setURLs(termsURL: URL, privacyPolicyURL: URL) {
         currentStatus = PersistenceManager.shared.retrieveStatus()
         self.termsURL = termsURL
         self.privacyPolicyURL = privacyPolicyURL
     }
-    
+
     public func setServices(services: [Service]) {
         guard let currentStatus = currentStatus else { return }
+        assert(!services.map { $0.id }.hasDuplicates, "The ids of the services have to be unique" )
         for service in services {
```

Comment:
I don't get this for loop... and how do I get rid of a service completely?

Reply:
this is nor part of the pr - we will have to refactor some stuff here